libmscore/score.cpp

```diff
@@ -392,24 +392,25 @@ Score::~Score()
 
 Score* Score::clone()
 {
```

Comment:
Could you explain literallly in few sentences what was wrong with the previous implementation?

Reply:
The previous implementation was fully copied from MasterScore::clone. It created MasterScore instance and returned pointer to it. Hence Notation stored pointer to MasterScore and used overrided methods from MasterScore, not from Score